compiler/rustc_codegen_llvm/src/gotoc/cbmc/goto_program/symtab_transformer/gen_c_transformer/expr_transformer.rs

```diff
@@ -0,0 +1,277 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use std::ops::{BitAnd, Shl, Shr};
+
+use super::super::super::super::MachineModel;
+use super::super::super::{
+    BinaryOperand, CIntType, Expr, Location, Parameter, Stmt, Symbol, SymbolTable, SymbolValues,
+    Type,
+};
+use super::super::Transformer;
+use super::common::type_to_string;
+use num::bigint::BigInt;
+use rustc_data_structures::fx::FxHashMap;
+
+/// Create an expr from an int constant using only values <= u64::MAX.
```

Comment:
This is needed because gcc allows 128 bit int variables, but not 128 bit constants

Reply:
Done.